gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -38,16 +38,13 @@ var gpii = fluid.registerNamespace("gpii");
             }
         },
         members: {
-            activeSessions: {}
+            activeSession: {},
+            activeUserTokens: []
```

Comment:
This state "activeUserTokens" is unnecessary - it can be inferred from the keys for activeSessions as was originally designed.